css/styles.css

```diff
@@ -0,0 +1,16 @@
+@media (max-width: 75em) {
+  html {
+    font-size: 60%;
+  }
+}
+.nav {
```

Comment:
please leave one blanc line between different  brackets to make the source code more lisible 🤝

Reply:
Kudos 👍  for mentioning the reasons why changes are needed.

Reply:
[OPTIONAL]Kindly ensure that you use the correct English syntax and gramma so that the other party understands. Please correct the word `lisible`